set.mm

```diff
@@ -131992,32 +131992,30 @@ the existence of its supremum (see ~ suprcl ).  (Contributed by Paul
 
     ${
       $d A w x y z $.  $d B w x y z $.
-      $( The closure of one end of an open real interval.  (Contributed by
-         Glauco Siliprandi, 11-Dec-2019.) $)
-      snunioo2 $p |- ( ( A e. RR* /\ B e. RR* /\ A < B ) -> ( ( A (,) B ) u. { B
-       } ) = ( A (,] B ) ) $=
+      $( The union of an open interval with its upper endpoint is a left-open
```

Comment:
I don't understand this.  Of the two original versions ~ ioounsn and ~ snunioo2, ~ ioounsn was 1 byte shorter and committed earlier to the database.  The minimization applied to ~ snunioo2 had an adverse effect, the proof appears longer than even both of the originals.

~ ioounsn2 should have been preferred, perhaps renamed if its name is less suggestive.

Reply:
I think I did choose the shorter proof (Glauco's), with 451 versus 454 bytes, and then further minimized it from 451 down to 447 bytes):
Here is what I get:
Current `develop` before this PR:
```
MM> sh pr ioounsn/C
Proof of "ioounsn":
---------Clip out the proof below this line to put it in the source file:
      ( vx vy vz vw cxr wcel clt wbr w3a cioo co cicc cun cioc 3ad2ant2 3adant3
      wceq cle csn iccid uneq2d wa simpl simpr 3jca simp3 xrleid df-ioo xrlenlt
      df-icc cv df-ioc wi xrltle adantrd xrltletr ixxun syl12anc eqtr3d ) AGHZB
      GHZABIJZKZABLMZBBNMZOZVFBUAZOABPMZVEVGVIVFVCVBVGVISVDBUBQUCVEVBVCVCKZVDBB
      TJZVHVJSVBVCVKVDVBVCUDVBVCVCVBVCUEVBVCUFZVMUGRVBVCVDUHVCVBVLVDBUIQCDEFABB
      NPIITTLITCDEUJCDEULBFUMZUKCDEUNVNGHZVCVCKVNBIJZVNBTJZVLVOVCVPVQUOVCVNBUPR
      UQABVNURUSUTVA $.
---------The proof of "ioounsn" (454 bytes) ends above this line.
MM> sh pr snunioo2/C
Proof of "snunioo2":
---------Clip out the proof below this line to put it in the source file:
      ( vx vy vz vw cxr wcel clt wbr w3a cioo co cicc cun cioc wceq syl cle wa
      csn simp2 iccid uneq2d simp1 simp3 xrleid df-ioo df-icc cv xrlenlt df-ioc
      simpl1 simpl2 simprl xrltled ex xrltletr ixxun syl32anc eqtr3d ) AGHZBGHZ
      ABIJZKZABLMZBBNMZOZVFBUAZOABPMZVEVGVIVFVEVCVGVIQVBVCVDUBZBUCRUDVEVBVCVCVD
      BBSJZVHVJQVBVCVDUEVKVKVBVCVDUFVEVCVLVKBUGRCDEFABBNPIISSLISCDEUHCDEUIBFUJZ
      UKCDEULVMGHZVCVCKZVMBIJZVLTZVMBSJVOVQTVMBVNVCVCVQUMVNVCVCVQUNVOVPVLUOUPUQ
      ABVMURUSUTVA $.
---------The proof of "snunioo2" (451 bytes) ends above this line.
```
and this PR has:
```
MM> sh pr ioounsn/C
Proof of "ioounsn":
---------Clip out the proof below this line to put it in the source file:
        ( vx vy vz vw cxr wcel clt wbr w3a cioo co cicc cun csn cioc wceq cle
        wa simp2 iccid syl uneq2d simp1 xrleidd df-ioo df-icc cv xrlenlt df-ioc
        simp3 simpl1 simpl2 simprl xrltled ex xrltletr ixxun syl32anc eqtr3d )
        AGHZBGHZABIJZKZABLMZBBNMZOZVFBPZOABQMZVEVGVIVFVEVCVGVIRVBVCVDUAZBUBUCUD
        VEVBVCVCVDBBSJZVHVJRVBVCVDUEVKVKVBVCVDULVEBVKUFCDEFABBNQIISSLISCDEUGCDE
        UHBFUIZUJCDEUKVMGHZVCVCKZVMBIJZVLTZVMBSJVOVQTVMBVNVCVCVQUMVNVCVCVQUNVOV
        PVLUOUPUQABVMURUSUTVA $.
---------The proof of "ioounsn" (447 bytes) ends above this line.
> sh pr ioounsnOLD/C
Proof of "ioounsnOLD":
---------Clip out the proof below this line to put it in the source file:
        ( vx vy vz vw cxr wcel clt wbr w3a cioo cicc cun cioc wceq 3ad2ant2 cle
        co 3adant3 csn iccid uneq2d simpl simpr 3jca simp3 xrleid df-ioo df-icc
        wa cv xrlenlt df-ioc wi xrltle adantrd xrltletr ixxun syl12anc eqtr3d )
        AGHZBGHZABIJZKZABLSZBBMSZNZVFBUAZNABOSZVEVGVIVFVCVBVGVIPVDBUBQUCVEVBVCV
        CKZVDBBRJZVHVJPVBVCVKVDVBVCUKVBVCVCVBVCUDVBVCUEZVMUFTVBVCVDUGVCVBVLVDBU
        HQCDEFABBMOIIRRLIRCDEUICDEUJBFULZUMCDEUNVNGHZVCVCKVNBIJZVNBRJZVLVOVCVPV
        QUOVCVNBUPTUQABVNURUSUTVA $.
---------The proof of "ioounsnOLD" (454 bytes) ends above this line.
```

Reply:
1. The original proof lengths shown in commit 1 (proof bytes only) were ioounsn 3 lines + 19 bytes, snunioo2 3 lines + 20 bytes.  This means, the credits go to Jon Pennant, 8-Jun-2019, at the time of this PR.
2. The same counting scheme applied to the shortened proof is 3 lines, 21 bytes.

Reply:
The excerpts above are the counts from `metamath.c`. If I remove the tag `(Proof shortened by Glauco Siliprandi, 11-Dec-2019.)` it's ok ?

Reply:
PS: did you take into account that proofs are indented differently ?

Reply:
From file:///home/wolf/metamath/web/conventions.html:
`In borderline cases, we typically place more importance on the number of compressed proof steps ...`.

I see that we have two competing metrics here, but I think, we can decide this case.

Reply:
OK, the indentation is in favour of the shortened proof.  But this holds only, if my second objection is considered not sufficient.  In any case, the credits got to Jon Pennant.

Reply:
Counting the essential bytes of the proofs, I get 271 (snunioo2), 264 (ioounsn), 268 (ioounsnOLD). Therefore, i will accept this PR. We should not struggle about some bytes here: most important is to remove the duplicate.